modules/plugin/geotiff/src/test/java/org/geotools/gce/geotiff/GeoTiffReaderTest.java

```diff
@@ -1150,4 +1151,34 @@ public void testFileDoesNotExist() throws Throwable {
             file.setReadable(true);
         }
     }
+
+    /**
+     * The GeoTiffReader should be able to read from an InputStream
+     */
+    @Test
+    public void testCanReadInputStream() throws IOException {
+        File rasterfile = TestData.file(GeoTiffReaderTest.class, "geo.tiff");
+        FileInputStream is = new FileInputStream(rasterfile);
```

Comment:
The stream should be closed at the end of the test (best to use a try/with/resources)